### PR TITLE
Spacing between dropdowns / radios

### PR DESCRIPTION
**Pull request by michalburger1**
Adding some basic margin to the country/date dropdowns and scale radio buttons.

### CODE REVIEW

**Comment by michalburger1** [comment]
Before:

before

**Comment by michalburger1** [comment]
After:

after

File: covid_graphs/covid_graphs/scripts/country_dashboard.py [hunk 1]
```diff
@@ -71,17 +71,19 @@ def create_dashboard(
                 if event != OTHER
             ],
             value=BK_20200411.name,
-            style={"width": "40%"},
+            style={"width": "220px", "margin": "4px 0px"},
```
Comment (lukipuki) [inline covid_graphs/covid_graphs/scripts/country_dashboard.py:74]: Are pixels the recommended unit even on very high definition displays? E.g. on a 4K?

And I actually prefer even higher margin, perhaps 6 pixels.

**Review by lukipuki — APPROVED** [review]
It seems that we found our UI expert!

File: covid_graphs/covid_graphs/scripts/country_dashboard.py (hunk 1 above)
Comment (michalburger1) [reply covid_graphs/covid_graphs/scripts/country_dashboard.py:74]: `px` doesn't actually mean pixels, it means "device independent pixels". They're defined in terms of viewing angle and they're typically implemented as 1 physical pixel on medium resolution screens and 2 physical pixels on high resolution screens.

Comment (lukipuki) [reply covid_graphs/covid_graphs/scripts/country_dashboard.py:74]: TIL!

**Pull request merged**